lesson04/assignment/css/_components.content.scss

```diff
@@ -0,0 +1,112 @@
+/*https://codepen.io/Digitango/pen/YQQBXR pulsing animation source code */
+
+.fa-heart-o {
+  color: $red;
+  padding:.5em;
+  animation: $a-heart;
+  cursor: pointer;
+}
+
+//no pulsing when clicked on
+.fa-heart {
+  transform: scale(2);
+  padding:.5em;
+  color: $red;
+  cursor: pointer;
+}
+
+//no pulsing when hovered - larger heart
+.heart:hover {
+  transform: scale(2);
+  animation:'';
+  cursor: pointer;
+}
+
+img{
+  width: 100%;
+  overflow: hidden;
+  margin: auto;
+}
+
+.card {
+  $self: &;
+  width: 33%;
+  position: relative;
+}
+
+
+/*https://www.w3schools.com/howto/howto_css_flip_card.asp card flipping source code*/
+
+.card.is-flipped .card__face{
+   transform: rotateY(180deg);
+}
+
+.card__face {
+    $self: &;
+    //height:auto;
+    text-align: center;
+    transition: transform 1.5s;
+    transition-timing-function: linear;
+    transform-style: preserve-3d;
+    //box-shadow: 3px -3px 3px #aaa;
+
+    &--front, &--back {
+      position: absolute;
+      overflow:hidden;
+      width:100%;
+      backface-visibility: hidden;
+      height:auto;
+    }
+
+
+    &--back {
+      transform: rotateY(180deg);
+      background: $gray;
+      //height:auto;
+      //height:100%;
+    }
+  }
```

Comment:
Here and above, no need to leave commented out code. It can be removed, and if you ever want to see what it was, it'll be visible in the commit information here on Github.